config/vufind/GetThis.yaml

```
@@ -0,0 +1,96 @@
+---
+# This file contains the settings for the GetThis feature
+
+# Either to display the copy number (next to the call number), default false
+showCopyNumber: true
+
+templates:
+  # Elements here correspond to a template file within templates/get-this with .phtml being stripped from the end
+  # The order changes the order templates are displayed
+  # Elements parameter, all parameters are optional :
+  #   enabled: true/false, whether to enable the template
+  #   view_variables: array of extra variables made available to the templates.
+  #     It can be used with placeholders if available; see example below.
+  #   condition_function: a function within GetThisLoader returning a boolean to whether display or not the template
```

Minor clarification:

```suggestion
  #   condition_function: a function within GetThisLoader returning a boolean indicating whether to display the template
```